macros.in

```diff
@@ -1120,6 +1122,23 @@ done \
 %{__patch} %{-p:-p%{-p*}} %{-q:-s}\
 %{__bzr} commit %{-q} -m %{-m*}
 
+# Subversion
+%__scm_setup_svn(q)\
+%{__svnadmin} create .svnrepos\
+%{__svn} mkdir %{-q} -m "Create directory structure." file://`pwd`/.svnrepos/trunk\
+%{__svn} checkout %{-q} file://`pwd`/.svnrepos/trunk ./\
+%{__svn} add %{-q} --force ./\
+%{__svn} commit %{-q} -m "Initial import." && %{__svn} update %{-q}
+
+# Subversion 1.6 doesn't have its own command to apply patches
+#%__scm_apply_svn(qp:m:)\
```

Comment:
Personally, I'ld vote for including the 'old' version of `%__scm_apply_svn` for _all_ versions of subversion (using standard `patch` instead of `svn patch`). `svn patch` does _not_ support pipe input, so compressed patches – rare as they might be – are completely unsupported by the 'new' version of `%__scm_apply_svn`. (I have no idea if the AS project is willing to extend `svn patch` for piped input; the [relevant source code](https://github.com/apache/subversion/blob/trunk/subversion/libsvn_client/patch.c#L3725) looks quite inflexible in this respect.)